runner/runner.go

```diff
@@ -58,19 +56,29 @@ const (
 	defaultProcessesCacheSize   = 16384
 )
 
-func startTraceHandling(ctx context.Context, rep otelreporter.TraceReporter,
-	intervals *times.Times, trc *tracer.Tracer, cacheSize uint32) error {
+func startTraceHandling(ctx context.Context, trc *tracer.Tracer) error {
 	// Spawn monitors for the various result maps
 	traceCh := make(chan *host.Trace)
 
 	if err := trc.StartMapMonitors(ctx, traceCh); err != nil {
 		return fmt.Errorf("failed to start map monitors: %w", err)
 	}
 
-	_, err := tracehandler.Start(ctx, rep, trc.TraceProcessor(),
```

Comment:
Need to check if it's worth disabling the frame cache that replaces the trace cache.